requirements.txt

```diff
@@ -0,0 +1,9 @@
+autobahn==0.18.2
+beautifulsoup4==4.5.3
+bs4==0.0.1
+-e git+git@github.com:sopython/rabbit.git@99cf02db5a983b7718712c194e39d9fdd55ec0c1#egg=rabbit
```

Comment:
Uh, not sure if this is supposed to be here. I've never actually used `requirements.txt` before.